go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent none
+    stages {
+        stage('TestSQL') {
+            parallel {
+                stage ("update Liquidata/dolt-sql-performance") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        DOLT_VERSION = sh(returnStdout: true, script: "(git rev-parse HEAD | tr -d '\n')")
+                        TMPDIR = "${pwd()}/tempDir"
+                        CREDSDIR = "/home/jenkins/.dolt/creds"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        TEST_N_TIMES = 5 // like to pass this from the Jenkins UI?
```

Comment:
Specifies the number of times we want to run the entire test suite... I'd like to be able to set this var using the jenkins UI, but am unsure how to set that up at the moment.